### PR TITLE
Temporary cron to move realtime data into watched RT incoming directo…

### PR DESCRIPTION
**Pull request by bpasquer**
…rywq
Needs to be reviewed and deploy shortly please

### CODE REVIEW

File: ANFOG/RT/move_to_incoming.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# script to loop through realtime subdirectories to find new netcdf files
+# then zip related files together
+INCOMING_RT=$INCOMING_DIR/ANFOG/realtime
+
+for file in `find $INCOMING_RT/. -iname "*FV0*.nc"`; do
```
Comment (mhidas) [inline ANFOG/RT/move_to_incoming.sh]: This is looping through *netcdf files*, not *subdirectories*. Is that what is intended?

File: ANFOG/RT/move_to_incoming.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+     fileNc=$file
+     zipName=`dirname $file`/"`basename $file .nc`.zip"
```
Comment (mhidas) [inline ANFOG/RT/move_to_incoming.sh]: I'm not sure what this is trying to do? is `zipName` supposed to be the file name with ".zip" appended?

**Comment by mhidas** [comment]
@bpasquer Should be deployed at 5 tonight.

**Comment by bpasquer** [comment]
thanks @mhidas

File: ANFOG/RT/move_to_incoming.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# script to loop through realtime subdirectories to find new netcdf files
+# then zip related files together
+INCOMING_RT=$INCOMING_DIR/ANFOG/realtime
+
+for file in `find $INCOMING_RT/. -iname "*FV0*.nc"`; do
+
+     fileNc=$file
```
Comment (danfruehauf) [inline ANFOG/RT/move_to_incoming.sh]: The indent... ;)

**Comment by lbesnard** [comment]
> The indent... ;)
>  Add a line note

:+1:

**Pull request merged**